dev.lapce.lapce.metainfo.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
```

Comment:
Same as the desktop file

Reply:
Thanks for your fast review! I'll create a PR for these files upstream.

Reply:
Are `<release>` tags mandatory for a release on flathub? `flatpak run org.freedesktop.appstream-glib validate dev.lapce.lapce.metainfo.xml` does complain if they are not present.
The upstream developers doesn't even use Linux, so I don't want to bother them with updating metainfo.xml with every release.

Reply:
yes it's required.

Reply:
The files have been merged into the upstream master branch an can be deleted after the next release.

Reply:
done